myuw/templates/handlebars/card/bothell_application.html

```diff
@@ -0,0 +1,66 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "bothell_application_card" %}
+<div class="myuw-card application-card">
+{{#if is_returning}}
+    <h3 class="myuw-card-title">
+        Your Returning Student Application
+    </h3>
+
+    <div class="myuw-card-section last">
+		<h4 class="myuw-card-heading myuw-header-msg"> For application status, contact the UW Bothell Office of Admissions</h4>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Email</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content"><a href="mailto:uwbifo@uw.edu">uwbifo@uw.edu</a></div>
+                </div>
+            </div>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Phone</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(425) 352-5000</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Mail</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">10909 NE 185th St <br>Bothell, WA 98011<br>Box 358500 </div>
+                </div>
+            </div>
+	</div>
+	<div class="myuw-card-section" style="padding-top: 8px;">
+		<div>
+			<h4 class="myuw-card-heading"> Resources for Bothell Applicants</h4>
+				<ul class="unstyled-list list-leading ">
+				    <li><a href="http://www.uwb.edu/financialaid" target="_blank" data-linklabel="Student Financial Aid">Learn all about student financial aid and scholarships</a></li>
+				    <li><a href="http://www.uwb.edu/studentaffairs" target="_blank" data-linklabel="Student Resources">Check out the student resources for student life, career services, and more</a></li>
+				   	<li><a href="http://www.uwb.edu/calendars" target="_blank" data-linklabel="UW Bothell Academic calendars">View the UW Bothell Academic calendars</a></li>
+				</ul>
+		</div> <!-- .links-container -->
+
+    </div>
+
+{{else}}
+    <h3 class="myuw-card-title">
+        Your Bothell Application for {{toTitleCase quarter}} {{year}}
+    </h3>
+
+    <div class="myuw-card-section">
+
+        <div class="clearfix">
+             <span class="btn btn-default card-action-btn myuw-btn-big"><a href="https://admissions.uwb.edu/apply/status"  data-linklabel="Your application status details">View your first year application status and checklist</a></span>
```

Comment:
For reference, here is where I'm seeing the two btn classes used together.

`<span class="btn btn-default card-action-btn myuw-btn-big">`